### ISSUE TITLE
ion-searchbar attribute "color" does not work under certain conditions on iOS

### ISSUE THREAD

**Issue by robingenz** [ionic-team/ionic#17186]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

The color attribute of the ion-searchbar component no longer works in iOS mode if ion-searchbar is child of ion-toolbar and ion-toolbar has also set the color attribute. 

**Expected behavior:**

The color attribute should change the color of ion-searchbar.

**Steps to reproduce:**

Create an ion-toolbar with ion-searchbar inside it and add the color attribute to both elements.

**Related code:**

```
<ion-toolbar color="primary">
    <ion-searchbar color="light"></ion-searchbar>
</ion-toolbar>
```

Example: https://github.com/robingenz/ionic-issue-17186-example

**Other information:**

**Ionic info:** 

```
Ionic:

   ionic (Ionic CLI)             : 4.8.0 (C:\Users\robin.genz\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.0.0-rc.1
   @angular-devkit/build-angular : 0.11.4
   @angular-devkit/schematics    : 7.1.4
   @angular/cli                  : 7.1.4
   @ionic/angular-toolkit        : 1.2.2

System:

   NodeJS : v10.15.0 (C:\Program Files\nodejs\node.exe)
   npm    : 6.5.0
   OS     : Windows 10
```

**Comment by brandyscarney** [comment]
Thanks for the issue! Confirmed this: https://codepen.io/brandyscarney/pen/ZwrbwR

**Comment by liamdebeasi** [comment]
Resolved via #17474
